libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -297,7 +283,32 @@ namespace eosio::testing {
             uint64_t                    threshold {0};
             std::vector<account_name>   local_finalizers;
          };
-         std::pair<transaction_trace_ptr, std::vector<fc::crypto::blslib::bls_private_key>> set_finalizers(const finalizer_policy_input& input);
+
+         struct set_finalizers_output_t {
+            transaction_trace_ptr        setfinalizer_trace;
```

Comment:
Seems should just be `trace` since it is in struct `set_finalizers_output_t`

Reply:
In general, I would agree, but because `set_finalizers()` does other things, I actually think it is clearer to specify what the trace is of. And Gnome already uses this branch I believe.

Reply:
It is the trace for the [setfinalizer](https://github.com/AntelopeIO/spring/blob/92a21962db0144c2174e1f66d308f72123382f08/libraries/testing/tester.cpp#L1254-L1256) action.